exercises/space-age/.meta/generator/space_age_cases.rb

```diff
@@ -0,0 +1,46 @@
+class SpaceAgeCase < ExerciseCase
+  SECONDS_PER_YEAR_ON_EARTH = 31557600
+
+  def workload
+    [
+      "age = SpaceAge.new(#{underscore_format(seconds)})",
+      indent(4, assertion),
```

Comment:
There are some (currently undocumented) methods in `ExerciseCases` that can do underscore formatting and indenting for you.

https://github.com/exercism/xruby/blob/master/lib/generator/exercise_cases.rb

https://github.com/exercism/xruby/blob/master/lib/generator/underscore.rb